src/contracts/interfaces/IPaymentCoordinator.sol

```diff
@@ -33,48 +32,73 @@ interface IPaymentCoordinator {
     }
 
     struct DistributionRoot {
-        // timestamp after which the root can be claimed against
-        uint64 activatedAfter;
         // merkle root of the distribution
         bytes32 root;
+        // The timestamp which calculated payments started
+        uint64 paymentCalculationStartTimestamp;
+        // The timestamp until which payments have been calculated
+        uint64 paymentCalculationEndTimestamp;
+        // timestamp at which the root can be claimed against
+        uint64 activatedAt;
     }
 
     struct ClaimsTreeMerkleLeaf {
```

Comment:
should we literally do a TokenTreeMerkleLeaf and a EarnerTreeMerkleLeaf?

Reply:
Aligns best with tree design IMO

Reply:
https://github.com/Layr-Labs/eigenlayer-contracts/pull/488/files#diff-a9495834f8b68265d0d625d94e217cace2813c0c4334cdaa645f8ae6e7e5e5bcR48